src/lib/config.js

```diff
@@ -27,7 +27,8 @@ module.exports = {
   HANDLERS_TIMEOUT_TIMEZONE: RC.HANDLERS.TIMEOUT.TIMEZONE,
   KAFKA_CONFIG: RC.KAFKA,
   PARTICIPANT_INITIAL_POSITION: RC.PARTICIPANT_INITIAL_POSITION,
-  RUN_MIGRATIONS: RC.RUN_MIGRATIONS,
+  RUN_MIGRATIONS: !RC.MIGRATIONS.DISABLED,
```

Comment:
why do you have a ! infront of RC.MIGRATIONS.DISABLED??

Reply:
It is negated: RUN_MIGRATION = true when RC.MIGRATIONS.DISABLED is false.